packages/travix-ui-kit/tests/unit/stepper/stepper.componentWillReceiveProps.spec.js

```diff
@@ -0,0 +1,82 @@
+import Stepper from '../../../components/stepper/stepper';
+
```

Comment:
Please, let not test React lifecycles. 

Jest and Enzyme (or the renderer used by UI Kit) are good enough to handle all props variations and expectations.

Reply:
In my point of view, testing each non-render method as a single isolated method is better for unit testing. That is the idea of unit testing: take smallest independent part and test it within isolated environment. Also if we decide to get rid of Jest screenshots in future, it will be much simpler to move non screenshots to new framework.